### DISCUSSION TITLE
Tabs in toolbar impossible to center

### DISCUSSION THREAD

**Discussion by Claus1** [quasarframework/quasar#11084] [CLI - PWA mode] [answered]
I need to center tabs in a toolbar without removing its title.
![image](https://user-images.githubusercontent.com/1247062/138566243-066607b2-6bc9-477e-bf0e-fe16d3126950.png)

https://codepen.io/Claus1/pen/RwZKgZR?editors=101

I tried align='center', style='float:left', .. nothing helps. The similar examples in toolbar docs are the same (right align) . How to?

**Comment by pdanpdan** [comment]
Centered in what? I'm the space between the end of title and right margin?

**Comment by Claus1** [comment]
@pdanpdan  Yes.

**Reply by hawkeye64** [reply]
At the end of the `q-tabs`, add a `q-space`

**Reply by JokerMartini** [reply]
Seems like he hasn't used the framework very much if he is so quick to say it's impossible.

**Comment by hawkeye64** [comment, marked as answer]
@Claus1 Here is a CodePen with QSpace: https://codepen.io/Hawkeye64/pen/BadWQKe?editors=1010
and an image: 
![image](https://user-images.githubusercontent.com/10262924/138697633-3b40f28d-bfe3-4b34-be45-848e63f7dcc6.png)

**Reply by Claus1** [reply]
Got it. Thanks!
